website/search/elastic_search.py

```diff
@@ -85,6 +85,9 @@ def wrapped(*args, **kwargs):
                 if 'ParseException' in e.error:
                     raise exceptions.MalformedQueryError(e.error)
                 raise exceptions.SearchException(e.error)
+            except TransportError as e:
+                # Catch and wrap generic ES 500 codes. TODO: Improve fix for https://openscience.atlassian.net/browse/OSF-4538
```

Comment:
Well, greater than 400, anyway. Will tweak the comment.

https://github.com/elastic/elasticsearch-py/blob/1.3.0/elasticsearch/exceptions.py#L27